### ISSUE TITLE
Validate requested spreadsheet name

### ISSUE THREAD

**Issue by rugby-player** [audreyt/ethercalc#743]
Audrey,

Great product - extremely impressed.

My environment is CentOS 8, Nginx, Redis. Nginx using auth_basic & auth_basic_user_file
I have a use case that I’m trying to figure out.

This is not an issue with the product is advice on how to protect us from dumb users.

We have setup a few collaborative spreadsheets for our various teams of the form.
https://hostname.com/ss00
https://hostname.com/ss01
https://hostname.com/ss02

We have the issue where some team members can’t type the URL correctly.
And they get really confused when they type something like
https://hostname.com/ss000
and get the splash screen for a new empty spreadsheet.

Basically what we are looking for is to be able to validate the spreadsheet name.
Such that I can say the only valid request_uri is in the list ["ss00","ss01","ss02]
If its not in the list we want to send back a 400 series response.

I figure we are not the first to want something like this.
What is your recommended approach to this?

Thank you in-advance.

Regards,

@rugby-player

**Comment by eddyparkinson** [comment]
try:

https://hostname.com/_rooms
https://hostname.com/_roomlinks

if depends on your setting. i.e.
  https://ethercalc.net/_rooms  
Gives "_roomlinks not available with CORS"
